docs/building/building.md

```diff
@@ -45,6 +45,8 @@ Copy your demo VHD or VHDX image to your Windows Machine and boot it with Hyper-
 1. Select _Settings...._
 1. Select Security and under _Template:_ select _Microsoft UEFI Certificate Authority_.
 
+*Note:* If you followed any of the previous tutorials in [Working with packages](docs/packages/working_with_packages.md), the kernel is no longer signed and secure boot must be disabled in order to boot your VHD(X) image.
```

Comment:
This is the biggest thing I want to call out. Do we think this is a thorough enough explanation?

I will copy and paste Suresh's comment on this: "I think when users modify and rebuild kernel RPM package, they might need to remove "lockdown=integrity" in grub cfg apart from secure boot disabling. This might be needed until we provide a way to external users to sign kernel modules trusted keys." 

But I do not think that "lockdown=integrity" applies to the tutorial repo. Please correct me if I am wrong.

Reply:
Regarding Secure Boot, I suggest giving adding precise steps by adding in something like "To disable Secure Boot, in the Security menu, uncheck the "Enable Secure Boot" box"

Regarding kernel lockdown, in this specific case it should not be required to remove "lockdown=integrity". Reason being that, during the kernel RPM build, the kernel will generate an ephermeral key and sign all of its kernel modules with this key. The newly built kernel will then inherently trust these modules.

Suresh's statement is true if you expect users to build their own custom kernel modules. In which case, we'll want to add instructions on how to remove "lockdown=integrity" from grub.cfg.